extensions-core/lookups-cached-single/src/test/java/org/apache/druid/server/lookup/LoadingLookupTest.java

```diff
@@ -136,13 +157,16 @@ public void testGetCacheKey()
   @Test
   public void testSupportsAsMap()
   {
-    Assert.assertFalse(loadingLookup.supportsAsMap());
+    Assert.assertTrue(loadingLookup.supportsAsMap());
   }
 
   @Test
   public void testAsMap()
   {
-    expectedException.expect(UnsupportedOperationException.class);
-    loadingLookup.asMap();
+    Map.Entry<String, String> fetchedData = new AbstractMap.SimpleEntry<>("dummy", "test");
```

Comment:
You could use `final Map<String, String> fetchedData = new HashMap<>();`.  Please include a couple more key-value pairs, one with null key and another with null value. If someone changes the underlying implementation of `asMap()`, this test will verify that the null cases are still handled.